slider/replay.py

```diff
@@ -38,8 +40,9 @@ class Action:
     mouse2 : bool
         is the second mouse button pressed?
     """
-    def __init__(self, offset, position, key1, key2, mouse1, mouse2):
-        self.offset = offset
+    def __init__(self, rel_offset, abs_offset, position, key1, key2, mouse1, mouse2):
+        self.rel_offset = rel_offset
```

Comment:
as a slight style nitpick, I would probably have called these `absolute_offset` and `relative_offset` so that there is no confusion.